composer.json

```diff
@@ -12,20 +12,20 @@
         },
         {
             "name": "Symfony Community",
-            "homepage": "http://symfony.com/contributors"
+            "homepage": "https://symfony.com/contributors"
         }
     ],
     "require": {
-        "php": ">=5.6",
-        "symfony/monolog-bridge": "~3.4 || ~4.0 || ^5.0",
+        "php": ">=7.1.3",
+        "symfony/monolog-bridge": "~4.0 || ^5.0",
         "symfony/dependency-injection": "~3.4.10 || ^4.0.10 || ^5.0",
-        "symfony/config": "~3.4 || ~4.0 || ^5.0",
-        "symfony/http-kernel": "~3.4 || ~4.0 || ^5.0",
+        "symfony/config": "~4.0 || ^5.0",
+        "symfony/http-kernel": "~4.0 || ^5.0",
         "monolog/monolog": "~1.22 || ~2.0"
     },
     "require-dev": {
-        "symfony/yaml": "~3.4 || ~4.0 || ^5.0",
-        "symfony/console": "~3.4 || ~4.0 || ^5.0",
+        "symfony/yaml": "~4.0 || ^5.0",
+        "symfony/console": "~4.0 || ^5.0",
         "symfony/phpunit-bridge": "^4.4 || ^5.0"
```

Comment:
`symfony/phpunit-bridge` should be changed to `^5.1`. there is no point in using an older version there, given that 5.1 supports all PHP versions we need